source/features/remove-upload-files-button.tsx

```diff
@@ -11,7 +11,7 @@ function init(): void {
 
 void features.add({
 	id: __filebasename,
-	description: 'Remove the "Upload files" button',
+	description: 'Remove the "Upload files" button.',
```

Comment:
Always in third person, as if it said “[this feature] hides the button”
```suggestion
	description: 'Hides the "Upload files" button.',
```

However I’d drop this feature altogether. It’s just a minor cleanup feature that only affects GHE now. New PR for this

Reply:
I don't think I will have time to go though all the descriptions now, maybe in a week or 2. For now I made  sure they all have periods.